src/util/address.rs

```diff
@@ -81,7 +81,9 @@ pub enum Error {
     /// An uncompressed pubkey was used where it is not allowed.
     UncompressedPubkey,
     /// Address size more than 520 bytes is not allowed.
-    ExcessiveScriptSize
+    ExcessiveScriptSize,
+    /// Script which is not a p2pkh, p2sh or witness program.
```

Comment:
```suggestion
    /// Script is not a p2pkh, p2sh, or witness program.
```

Reply:
Thanks @tcharding for your review and feedback! 
The link was super helpful and applied the takeaways from it to this PR. Kindly let me know how it is right now and where it could be further improved.